### PR TITLE
Fix testing and update README

### PR DESCRIPTION
**Pull request by sbialkowski-r7**
## Description

- README.md update

### CODE REVIEW

**Comment by sbialkowski-r7** [comment]
(get travis to work)

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,11 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
-  - 7.2
```
Comment (rjacobs-r7) [inline .travis.yml:10]: these versions may be EOL, but they're definitely still being used... i don't think we can remove any 7.x version for now, and keeping 5.6 around for a while is probably nice, too. i think we can drop support for them once we actually introduce new features that are not backwards-compatible (or typehints).

File: unit_tests/R7LoggerTests.php [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?php
 	use PHPUnit\Framework\TestCase;
 
-	class R7LoggerTest extends TestCase
+	class R7LoggerTests extends TestCase
```
Comment (rjacobs-r7) [inline unit_tests/R7LoggerTests.php:4]: iirc singular is convention. i don't particularly care but i don't see why we're changing this.

File: unit_tests/R7LoggerTests.php [hunk 3]
```diff
@@ -86,7 +86,7 @@ public function testGetAddress()
 		public function testEmptyToken() {
 			$this->expectException(InvalidArgumentException::class);
 
-			R7Logger::getLogger($this->definitely_a_valid_attribute, self::region, self::persistent, true, self::severity, self::datahubEnabled, self::datahubIPAddress, self::datahubPort, self::host_id, self::host_name, self::host_name_enabled, self::add_local_timestamp, self::use_json);
+			R7Logger::getLogger("", self::region, self::persistent, true, self::severity, self::datahubEnabled, self::datahubIPAddress, self::datahubPort, self::host_id, self::host_name, self::host_name_enabled, self::add_local_timestamp, self::use_json);
```
Comment (rjacobs-r7) [inline unit_tests/R7LoggerTests.php:89]: not sure if intentional, but this changes the test case from NULL to an empty string.

File: unit_tests/R7LoggerTests.php (hunk 2 above)
Comment (sbialkowski-r7) [reply unit_tests/R7LoggerTests.php:4]: Has to match the file name.
phpunit errors out otherwise

File: .travis.yml (hunk 1 above)
Comment (sbialkowski-r7) [reply .travis.yml:10]: Removed these because they crash on travis-ci.
I think it's because I bumped phpunit version, I'll drop it back down and see whether they run.

File: unit_tests/R7LoggerTests.php (hunk 3 above)
Comment (sbialkowski-r7) [reply unit_tests/R7LoggerTests.php:89]: Intentional, travis seems to crash on undefined attribute but my local doesn't.
Also this tests afaik tests the empty check we have, empty evaluates to true on 0, `""` so on so this to me is still a valid check
https://github.com/sbialkowski-r7/r7insight_php/blob/composer-add-readme/R7Logger.php#L155

Comment (rjacobs-r7) [reply unit_tests/R7LoggerTests.php:89]: ah, yes. let's develop a language that has a settings file that allows users to change the behaviour of the entire language. sounds like a wonderful idea.

Comment (rjacobs-r7) [reply unit_tests/R7LoggerTests.php:89]: as for equality checks; https://www.php.net/manual/en/types.comparisons.php#types.comparisions-loose
it depends on the operator used.
